controller/src/main/java/org/jboss/as/controller/management/Capabilities.java

```diff
@@ -40,4 +40,6 @@
 
     public static final String SSL_CONTEXT_CAPABILITY = "org.wildfly.security.ssl-context";
 
+    public static final String MANAGEMENT_SECURITY_REALM_CAPABILITY = "org.wildfly.core.management.security.realm";
```

Comment:
I have a fork of the capabilities repo with all of the Elytron capabilities defined, just needs an audit to double check recent changes have been applied and then it can be merged to the main repo.

https://github.com/wildfly-security-incubator/wildfly-capabilities